src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,30 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Please revisit checklist again: https://github.com/mate-academy/jv-oop-advanced/pull/849

Reply:
sorry, i don't understand, because in my code there is no "return null;"
and what should I look at in the link to another task?

Reply:
I'm sorry, I've pasted the wrong link

Reply:
github shows `return null` as deleted line (it's in default empty project), it's okay

Reply:
I meant to share [this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field)

Reply:
ok, thanks